modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/components/form-renderer-custom-fields/RequiredField.es.js

```diff
@@ -15,7 +15,6 @@
 import {ClayButtonWithIcon} from '@clayui/button';
 import ClayForm, {ClayRadio, ClayRadioGroup, ClayToggle} from '@clayui/form';
 import ClayPopover from '@clayui/popover';
-import {ClayTooltipProvider} from '@clayui/tooltip';
```

Comment:
I'd expect at least some removals in `package.json` dependencies of `@clayui/tooltip` that are surely unnecessary after this.

Reply:
Very true, didn't think about that. I'll update